src/components/Navbar/__test__/Navbar.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from "react-test-renderer";
+import Navbar from "../Navbar";
```

Comment:
Or we use always alias imports like so : `import NavBar from "@components/NavBar/NavBar"` instead of relative imports "import NavBar from "../NavBar"`
@Lilicod